src/core/utils.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-function getMantissaDecimalString(bignum) {
-  let mantissa = bignum.toPrecision(16)
-    .replace(/\./, '')      // remove decimal point
+// Should only be used directly when the string is from a BigNumber
```

Comment:
I think for our use cases we would probably want to truncate the least significant digits. We might want to add "16" to the name of the function as a reminder.